searchlib/src/tests/attribute/tensorattribute/tensorattribute_test.cpp

```diff
@@ -658,6 +667,69 @@ class DenseTensorAttributeMockIndex : public Fixture {
     DenseTensorAttributeMockIndex() : Fixture(vec_2d_spec, true, true, true) {}
 };
 
+TEST_F("blueprint takes global filter into account", DenseTensorAttributeMockIndex)
```

Comment:
For consideration: To make the tests easier to read I would prefer having one test function for each of the 3 cases tested, and move the common setup (tensor content + blueprint) to a fixture that could inherit DenseTensorAttributeMockIndex.